internal/util/validation/validation.go

```diff
@@ -107,6 +108,52 @@ func ValidateBearerToken(token *string, path string) []error {
 	return asErrors(errs)
 }
 
+// usages for a CSR should include 'client auth'
+// TODO - do we want to add more here? or just not let the user populate this?
+func ValidateUsages(u *[]string) []error {
+	errs := field.ErrorList{}
+
+	for _, a := range *u {
+		if a == "client auth" {
+			return nil
+		}
+	}
+	errs = append(errs, field.Invalid(fieldPathFor("spec.usages"), u, "must contain 'client auth'"))
+	return asErrors(errs)
+}
+
+// signer name from https://github.com/flightctl/flightctl/blob/75cb5bfada1f1dbd995797edb5d8a67ac251b861/cmd/flightctl-api/main.go#L26
+func ValidateSignerName(s string) []error {
+	errs := field.ErrorList{}
+	if s != "ca" {
+		errs = append(errs, field.Invalid(fieldPathFor("spec.signerName"), s, "must specify a valid signer"))
+		return asErrors(errs)
+	}
+	return nil
+}
+
+// TODO - should this just be a warning?
```

Comment:
Depends - what happens if it's less than 600?

Reply:
600 is an arbitrary limit. All of this should be pulled out from hard coded into config at a later date

Reply:
I've made it a warning for now. We can make it stricter later if it matters.